docs/latest/user/udp-routing.md

```diff
@@ -0,0 +1,59 @@
+# UDP Routing
+
+The [UDPRoute][] resource allows users to configure UDP routing by matching UDP traffic and forwarding it to
+Kubernetes backends. To learn more about UDP routing, refer to the [Gateway API documentation][].
+
+Follow the steps from the [Quickstart Guide](quickstart.md) to install Envoy Gateway and then install the example
+resources used for this guide.
```

Comment:
As with other docs, the QuickStart acts as a baseline. I suggest copying the language verbatim from the other docs, e.g. ensure the QuickStart examples work.